astropy/tests/runner.py

```diff
@@ -166,6 +178,10 @@ def _generate_args(self, **kwargs):
         """
 
     def run_tests(self, **kwargs):
+        if not _has_test_dependencies(): # pragma: no cover
+            msg = "Test dependencies are missing. Try 'pip install astropy[test]'"
```

Comment:
This error message is misleading since the user may not have installed astropy with pip - instead i think it should just say 'you should install the pytest-astropy package'.